### ISSUE TITLE
latest paseo.raw.json causes client to crash loop

### ISSUE THREAD

**Issue by hitchhooker** [paseo-network/runtimes#18]
updating to use latest paseo.raw.json client keeps crashing due to added lines

```
Jan 26 15:36:06 pc01 systemd[1]: polkadot.service: Main process exited, code=exited, status=1/FAILURE
Jan 26 15:36:06 pc01 systemd[1]: polkadot.service: Failed with result 'exit-code'.
Jan 26 15:36:16 pc01 systemd[1]: polkadot.service: Scheduled restart job, restart counter is at 8.
Jan 26 15:36:16 pc01 systemd[1]: Started "Rotko Networks - pso01 validator - Polkadot service".
Jan 26 15:36:16 pc01 polkadot[1771315]: Error:
Jan 26 15:36:16 pc01 polkadot[1771315]:    0: Invalid input: Error parsing spec file: failed to parse: Invalid multihash size 33. at line 9 column 4
Jan 26 15:36:16 pc01 polkadot[1771315]: Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Jan 26 15:36:16 pc01 polkadot[1771315]: Run with RUST_BACKTRACE=full to include source snippets.
Jan 26 15:36:16 pc01 systemd[1]: polkadot.service: Main process exited, code=exited, status=1/FAILURE
Jan 26 15:36:16 pc01 systemd[1]: polkadot.service: Failed with result 'exit-code'.
```
works
```default_chain_spec_dl_url: "https://raw.githubusercontent.com/paseo-network/runtimes/8cb10b1404f3e1ec79aa72f88d182f6a3f38b3a3/chain-specs/paseo.raw.json"```
latest doesnt
```#default_chain_spec_dl_url: "https://raw.githubusercontent.com/paseo-network/runtimes/main/chain-specs/paseo.raw.json"```

```
root@pso06:/opt/polkadot# diff paseo.raw.json paseo.raw.json.crash
7a8,9
>     "/dns/paseo-bootnode.radiumblock.com/tcp/30333/p2p/12D3KooWADeaC8zag4Qrb4GosSn65MmfVZztRPMaBdgZnQqXRo",
>     "/dns/paseo-bootnode.radiumblock.com/tcp/30335/wss/p2p/12D3KooWeayZC8zag4Qrb4GosSn65MmfVZztRPMaBdgZnQqXRo",
```

**Comment by al3mart** [comment]
Fix merged #20

**Comment by 09306677806** [comment]
Ok Tenkyu

در تاریخ دوشنبه ۲۹ ژانویه ۲۰۲۴،‏ ۰۷:۴۳ Alejandro Martinez Andres <
***@***.***> نوشت:
